playground/main.js

```diff
@@ -12,9 +12,8 @@ const renderPlaygroundWidget = (options = {}) => {
   if (signIn) {
     signIn.remove();
   }
-  // TODO: shall we use deep merge cause widget option has object value,
-  // which will be override when doing `Object.assign`.
-  signIn = new OktaSignIn(Object.assign(signinWidgetOptions, options));
+
+  signIn = new OktaSignIn(Object.assign({}, signinWidgetOptions, options));
```

Comment:
I think the TODO still relevant. It's the nested object didn't got copied, e.g. `option.features`